src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -73,14 +77,50 @@ public JobService(IamService samService,
 
         logger.info("Creating Stairway: maxStairwayThreads = " + appConfig.getMaxStairwayThreads());
         ExceptionSerializer serializer = new StairwayExceptionSerializer(objectMapper);
+        StairwayHook hooks = populateStairwayHooks();
         stairway = Stairway.newBuilder()
             .maxParallelFlights(appConfig.getMaxStairwayThreads())
             .exceptionSerializer(serializer)
             .applicationContext(applicationContext)
             .stairwayName(appConfig.getPodName())
+            .stairwayHook(hooks)
         .build();
     }
 
+    private StairwayHook populateStairwayHooks(){
+        return new StairwayHook() {
+            @Override
+            public HookAction startFlight(FlightContext context) {
+                logger.info("Operation: {}, flightClass: {}, flightId: {}, timestamp: {}",
+                    "startFlight", context.getFlightClassName(), context.getFlightId(), Instant.now());
```

Comment:
Is this the correct way to grab the current timestamp for the log?

Reply:
Looks right to me.

Reply:
We might want to specify a timezone and formatting, something like this:

```suggestion
                    "startFlight", context.getFlightClassName(), context.getFlightId(), Instant.now().atZone(ZoneId.of("Z")).format(DateTimeFormatter.ISO_INSTANT));
```